src/components/vedtak/VedtakInfoBox.tsx

```diff
@@ -14,24 +9,31 @@ const StyledColumn = styled(Column)`
 `;
 
 const StyledIcon = styled.img`
+  width: 5em;
   margin-top: 5em;
+  align-self: center;
 `;
 
 const grey = "#78706A";
 
-const StyledCenteredText = styled.h4`
+const StyledCenteredText = styled.h3`
```

Comment:
Kan vurdere å endre til riktig komponent fra https://design.nav.no/components/typografi i stedet for vanlig h3

Reply:
Litt usikker på hvilken komponent dette blir? Undertittel?